.circleci/config.yml

```diff
@@ -9,18 +9,109 @@ executors:
       xcode: 11.7
     environment:
       TERM: xterm-256color
-    
+
+aliases:
+  - &restore_macos_brew_cache
```

Comment:
why is this an alias? it's not reused, nor is any of its keys assigned when being referenced (<<: *). Perhaps the alias part could be the `when` condition, and the steps could be specified on reference.